PowerShellTools/DebugEngine/BreakpointManager.cs

```diff
@@ -145,14 +145,7 @@ public void SetBreakpoint(ScriptBreakpoint breakpoint)
 
             try
             {
-                if (Debugger.DebuggingService.GetRunspaceAvailability() == RunspaceAvailability.Available)
-                {
-                    Debugger.DebuggingService.SetBreakpoint(new PowerShellBreakpoint(breakpoint.File, breakpoint.Line, breakpoint.Column));
-                }
-                else if (Debugger.IsDebuggingCommandReady)
-                {
-                    Debugger.DebuggingService.ExecuteDebuggingCommandOutNull(string.Format(DebugEngineConstants.SetPSBreakpoint, breakpoint.File, breakpoint.Line));
-                }
+                Debugger.DebuggingService.SetBreakpoint(new PowerShellBreakpoint(breakpoint.File, breakpoint.Line, breakpoint.Column), Debugger.IsDebuggingCommandReady);

```

Comment:
As far as I can tell, this refactor is not necessarily the same as its original behavior, one difference is that if user add bp during scripting executing, your current implementation is going to try to execute it as debug command, which would end up exception on host service. Note that original behavior won't allow this request get forwarded to host service.